lib/views/details_page/add_to_list__button.dart

```diff
@@ -39,7 +43,13 @@ class AddToListButton extends StatelessWidget {
             message: addTooltip,
             child: IconButton(
               onPressed: () {
-                cubit.addGame(game);
+                if (context.read<AuthBloc>().state.authStatus !=
+                    AuthStatus.authenticated) {
+                  ScaffoldMessenger.of(context)
+                      .showSnackBar(MySnackBar(text: 'login_to_add'.tr));
```

Comment:
This text doesn't work.
<img width="327" alt="Zrzut ekranu 2022-08-8 o 20 06 53" src="https://user-images.githubusercontent.com/44871866/183484285-56c02cc8-dc97-47c1-845c-23f5ae8802b0.png">

Reply:
And add ','. ! :)